package.json

```diff
@@ -131,6 +132,16 @@
                     "light": "resources/light/folder.svg",
                     "dark": "resources/dark/folder.svg"
                 }
+            },
+            {
+                "command": "neo-visual-devtracker.createWalletFile",
+                "title": "Create new NEP-6 wallet file",
+                "category": "Neo Express"
```

Comment:
Create NEP-6 wallet + account commands shouldn't be in the Neo Express category. 

We probably need to review all of the command categories for things that are specific to neo-express vs. things that are not, but I'm fine doing that as a seperate PR

Reply:
https://github.com/neo-project/neo-visual-tracker/issues/50